pkg/api/norman/customization/oci/handler.go

```diff
@@ -191,14 +191,14 @@ func (handler *handler) extractCreds(req *http.Request, creds *Credentials) (int
 		// Get credentials from body
 		raw, err := ioutil.ReadAll(req.Body)
 		if err != nil {
-			logrus.Debugf("[oci-handler] cannot read request body: " + err.Error())
-			return httperror.InvalidBodyContent.Status, fmt.Errorf("cannot read request body: " + err.Error())
+			logrus.Debugf("[oci-handler] cannot read request body: %s", err)
```

Comment:
I always use `%v` for formatting errors, feel free to ignore this comment though.

Reply:
I guess it's a matter of preference. The `%v` means the default value, which is a string representation for the object implementing the `error` interface.

> If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).